app/gui2/src/components/lexical/index.ts

```diff
@@ -15,9 +16,36 @@ export interface LexicalPlugin {
   register: (editor: LexicalEditor) => void
 }
 
+export function lexicalTheme(theme: Record<string, string>): EditorThemeClasses {
+  function getTheme(className: string) {
+    if (!(className in theme))
+      console.warn(`Referenced class ${className} not found in lexical theme.`, theme)
+    return theme[className]!
```

Comment:
I would prefer returning some explicit "empty value" in case of lookup failure, and avoid using `!`.